### ISSUE TITLE
After updaing latest release vimeo link video not working...

### ISSUE THREAD

**Issue by thimfy** [CMB2/CMB2#1247]
After updating latest version of cmb2 vimeo video not working with its link, youtube or soundcloud working fine, but in same code vimeo video not working.

$cmb = new_cmb2_box( array(
		'id'            => $prefix . 'format_metabox',
		'title'         => __( 'Youtube, Vimeo, Audio Posts', 'theme' ),
		'object_types'  => array( 'post', ), // Post type
		'context'       => 'normal',
		'priority'      => 'high',
		'show_names'    => true, // Show field names on the left
		// 'cmb_styles' => false, // false to disable the CMB stylesheet
		// 'closed'     => true, // Keep the metabox closed by default
	) );
	
	$cmb->add_field( array(
		'name' => esc_html__( 'URL', 'theme' ),
		'desc' => 'Enter a youtube, vimeo or audio URL.',
		'id'   => $prefix . 'embed',
		'type' => 'oembed',
	) );
	}

Output: youtube, soundcloud working but vimeo not working...
$embed_url = esc_url( get_post_meta( get_the_ID(), '_theme_embed', true ) );
echo wp_oembed_get( $embed_url );

**Comment by tw2113** [comment]
Is it still outputting/working/playable on the frontend, regardless of the thumbnail preview on the admin side in the metabox itself?

Can you provide some of the exact urls of the videos you're saving?

**Comment by tw2113** [comment]
Note: I realize i can for 2 of them from the screenshots. But any others that are and aren't working as well

**Comment by thimfy** [comment]
These two are working with thubmnail in admin and frontend working
https://soundcloud.com/foxxtrax/shakira-remix-lala-v3-for-sound-cloud
https://www.youtube.com/watch?v=NHRq-8ek1NI

vimeo not working
https://vimeo.com/110258348
not in admin not in frontend in same code, even twitter also working only vimeo not woring

**Comment by thimfy** [comment]
as far i tested till not looks like only vimeo not working

**Comment by tw2113** [comment]
My initial hunches are something trickling down into WP core, but I haven't had a chance to test/confirm quite yet. It's in my todo list at the moment.

Thank you for all the information thus far, it will be helpful.

**Comment by tw2113** [comment]
As is, I'm not able to repeat with the provided vimeo video. I have used your code exactly, minus the $prefix variable, and the attached screenshot is what I'm getting for both backend and frontend. 
![cmb2-embed](https://user-images.githubusercontent.com/484075/55205552-60600200-51a1-11e9-9699-d80aa6e8fb62.png)

Possible that other things that I can't see are conflicting.

**Comment by thimfy** [comment]
In my case its saying: No oEmbed Results Found for https://vimeo.com/110258348. View more info at codex.wordpress.org/Embeds.
What could be cause of this? it is not able to recognize that this has any oEmbed 
http://i.share.pho.to/b0107653_o.png

**Comment by tw2113** [comment]
might be a case of it can't communicate out enough to get the remote data for the video. I know I was seeing some browser dev tools console logs around this specific topic, perhaps pop that open and see what results you're seeing come in.

**Comment by thimfy** [comment]
Not sure anything.
But in other theme where i am using ACF plugin same kind of thing working fine even in onother child theme of wordpress, custom iframe code with vimeo working fine, All working fine in local host.

**Comment by thimfy** [comment]
Did you check same in latest version of WP  Version 5.1.1 , could be core issue?

**Comment by tw2113** [comment]
Yep, the local install where I was testing has been on the latest WP core since it was released.

**Comment by thimfy** [comment]
Looks like problem with core, i have also tested it on latest version of wp and cmb2 and local host, theme Twenty Nineteen, and getting same issue.

**Comment by tw2113** [comment]
honestly i'd check the network tab for your browser since it's functionality that's going to make external requests as part of the oembed functionality. That tab may provide some extra insight. I'm also wondering if you have any sort of network traffic blocking of some sort that could be contributing.

Haven't seen any other reports of this type of issue, so for the moment at least, it's rather isolated. I do leave it open to in fact being a bug with CMB2 core, but not presently personally leaning towards that being the case.
